src/screens/AffiliationViews/DatabaseView/Schema.tsx

```diff
@@ -315,8 +315,8 @@ const Schema: React.FC<ISchemaProps> = ({
 };
 
 export default styled(Schema)`
-  height: 100%;
-  overflow-x: auto;
+  height: 97%;
```

Comment:
Åpen for alternative løsninger her. Den siste rowen i detailslist blir ikke vist med 100% height